### DISCUSSION TITLE
Single quote keyword table name

### DISCUSSION THREAD

**Discussion by misantroop** [duckdb/duckdb#9559] [General]
It was my understanding that double quotes are reserved for column names and everything else should use single quotes. However using a keyword for a table name with single quotes fails:

```
import duckdb
con = duckdb.connect('test.db')

# Fails
con.execute(f"""
     DROP TABLE IF EXISTS 'semi';
""")

# Succes!
con.execute(f"""
     DROP TABLE IF EXISTS "semi";
""")
```

**Comment by szarnyasg** [comment]
Hello, this comes up quite frequently. In shorts, single quotes should be used for regular strings and double quotes should be used for denoting table/column names (and escaping reserved keywords). They occasionally work in other roles but (e.g. `FROM "something.csv"` works) but their use is discouraged.
